src/engraving/dom/cmd.cpp

```diff
@@ -683,9 +683,10 @@ void Score::expandVoice(Segment* s, track_idx_t track)
     //
     Measure* m = s->measure();
     Fraction stick  = ps ? ps->tick() : m->tick();
+    Fraction stretch = staff(track2staff(track))->timeStretch(stick);
```

Comment:
A small thought about neatness - it might be worthwhile having a corresponding `actualTicks()` method wherever we have a `ticks()` method. This stretch calculation comes up often enough for segments.

Reply:
Good point. It's tricky to do for segments though, because a segment spans all staves, whereas the "stretch" (i.e. the ratio between the global and local time signature) is specific of one single staves. But I agree it'd be best to come up with something general, I'll do it in a next PR 👍